### PR TITLE
add support for setting custom logos in sunstone

### PR DESCRIPTION
**Pull request by jtriley**
This adds support for adding your own custom logo to sunstone:

```
one::oned::sunstone_logo_png: 'puppet:///modules/mymodule/front-page-logo.png'
one::oned::sunstone_logo_small_png: 'puppet:///modules/mymodule/top-left-logo.png'
```

### CODE REVIEW

File: manifests/oned/sunstone/config.pp [hunk 1]
```diff
@@ -27,6 +27,8 @@
   $vnc_proxy_cert        = $one::vnc_proxy_cert,
   $vnc_proxy_key         = $one::vnc_proxy_key,
   $vnc_proxy_ipv6        = $one::vnc_proxy_ipv6,
+  $logo_png              = $one::sunstone_logo_png,
```
Comment (tuxmea) [inline manifests/oned/sunstone/config.pp]: please keep variable names identical.
changing variable names across modules makes it more dificult to read and understand the code.

Comment (jtriley) [inline manifests/oned/sunstone/config.pp]: @tuxmea OK sure was just following the pattern already used in that class:

```
 $listen_ip = $one::sunstone_listen_ip,
```

**Comment by jtriley** [comment]
@tuxmea comments addressed. Should be good for merge.

**Comment by rwaffen** [comment]
waiting for the tests and will merge if all are passed...

**Comment by tuxmea** [comment]
OK for merge. @jtriley can you please update the in code documentation for the new parameter?

**Comment by jtriley** [comment]
@tuxmea Should be all set.

**Comment by tuxmea** [comment]
@jtriley great. thanks. @rwaffen feel free to merge after travis runs green.

**Pull request merged**